roles/openshift_master/handlers/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
-- name: restart openshift-master
-  service: name=openshift-master state=restarted
+- name: restart master
+  service: name=master state=restarted
```

Comment:
Sorry, not sure why my comment was detached from the code for some reason. This should be because the services are 'openshift-master' and 'atomic-enterprise-master' correct? Same with the 'restart node' I think we should also change 'known_openshift_product_types' to only include 'openshift' and 'atomic-enterprise'
```
  service: name={{ product_type}}-master state=restarted
```